src/psp2_ui.cpp

```diff
@@ -134,6 +138,18 @@ void Psp2Ui::ProcessEvents() {
 	trigger_state = (input.buttons & SCE_CTRL_RTRIGGER);
 	if ((trigger_state != old_state) && trigger_state) zoom_state = ((zoom_state + 1) % 3);
 	
+	// Left analog support
+	if (input.ly > 170) keys[Input::Keys::UP] = true;
+	else if (input.ly < 50) keys[Input::Keys::DOWN] = true;
+	else if (input.lx > 170) keys[Input::Keys::RIGHT] = true;
+	else if (input.lx < 50) keys[Input::Keys::LEFT] = true;
+	
```

Comment:
As this is technically a analog stick, you should define for VITA

```c++
#  define SUPPORT_JOYSTICK
#  define SUPPORT_JOYSTICK_AXIS
```
in `system.h` and use something like that:
```c++
	keys[Input::Keys::JOY_AXIS_X_LEFT] = (input.lx < 50);
	keys[Input::Keys::JOY_AXIS_X_RIGHT] = (input.lx > 170);
	keys[Input::Keys::JOY_AXIS_Y_DOWN] = (input.ly < 50);
	keys[Input::Keys::JOY_AXIS_Y_UP] = (input.ly > 170);
```